learners/starter-pack.md

```diff
@@ -1,7 +1,20 @@
----
-nav_order: 0
-title: Starter Pack
-parent: For Learners
----
-
-# Starter Pack
```

Comment:
We should probably keep the 'Starter Pack' title so it makes sense in the sidebar. Other headings should be level 2 or deeper (`##`) as it's best practice to only have one level 1 heading (`#`) on a page.